composer.lock

```diff
@@ -23195,5 +23193,5 @@
         "php": ">=7.4"
     },
     "platform-dev": [],
-    "plugin-api-version": "2.3.0"
+    "plugin-api-version": "2.2.0"
```

Comment:
I keep seeing this flip flop. Should we or should we not be on the latest plugin api version?